Sources/TSCBasic/Process.swift

```diff
@@ -731,11 +749,15 @@ private func open(pipe: inout [Int32]) throws {
 }
 
 /// Close the given fd.
-private func close(fd: inout Int32) throws {
-    let rv = TSCLibc.close(fd)
-    guard rv == 0 else {
-        throw SystemError.close(rv)
+private func close(fd: Int32) throws {
+    func innerClose(_ fd: inout Int32) throws {
```

Comment:
Is this just cleanup to avoid having to pass the file descriptor by address, or is there another reason for changing this function?

Reply:
yeah, did this in the beginning and decided to leave it in, can take it out if you're against it, but IMO it's cleaner use of the APIs.

Reply:
No, I was just curious if I was missing some semantic change.